tasks/files-to-granules/index.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const get = require('lodash.get');
+const path = require('path');
+
+const { getGranuleId } = require('@cumulus/cmrjs');
+const { parseS3Uri } = require('@cumulus/common/aws');
+const cumulusMessageAdapter = require('@cumulus/cumulus-message-adapter-js');
+
+/**
+ * Helper to turn an s3URI into a fileobject
+ * @param {string} s3URI s3://mybucket/myprefix/myobject.
+ * @returns {Object} file object
+ */
+function fileObjectFromS3URI(s3URI) {
```

Comment:
This is defined in both the `files-to-granules` task and the `move-granules` task.  Is it something that could be moved to somewhere common?

Reply:
It's very possible to refactor it out of move-granules. I've not yet had the time to fully commit but the only reason it's in there is to handle an array of CMR file URIs.

Reply:
5dfe0f087